src/build.py

```diff
@@ -919,7 +857,9 @@ def LLVM():
       '-DLLVM_TOOL_LTO_BUILD=OFF',
       '-DLLVM_INSTALL_TOOLCHAIN_ONLY=ON',
       '-DLLVM_ENABLE_ASSERTIONS=ON',
-      '-DLLVM_TARGETS_TO_BUILD=X86;WebAssembly'
+      '-DLLVM_TARGETS_TO_BUILD=X86;WebAssembly',
+      '-DLLVM_EXTERNAL_CLANG_SOURCE_DIR=%s' % CLANG_SRC_DIR,
+      '-DLLVM_EXTERNAL_LLD_SOURCE_DIR=%s' % LLD_SRC_DIR
```

Comment:
Why do you need these extra defines?

Reply:
Because the monorepo is flat (llvm, clang, and lld are all next to each other, instead of clang being checked out as llvm/tools/clang). So when you build LLVM the source is external rather than in the llvm tree.